ov/storage_volume.go

```diff
@@ -8,22 +8,33 @@ import (
 	"github.com/docker/machine/libmachine/log"
 )
 
+// The Marshal() will omit the bool attibutes below if they are false.
+// Please remove the omitempty option and use it as and when required.
+
 type StorageVolumeV3 struct {
-	Category               string                 `json:"category,omitempty"`
-	Created                string                 `json:"created,omitempty"`
-	Description            string                 `json:"description,omitempty"`
-	ETAG                   string                 `json:"eTag,omitempty"`
-	Name                   string                 `json:"name,omitempty"`
-	State                  string                 `json:"state,omitempty"`
-	Status                 string                 `json:"status,omitempty"`
-	Type                   string                 `json:"type,omitempty"`
-	URI                    utils.Nstring          `json:"uri,omitempty"`
-	Shareable              bool                   `json:"shareable,omitempty"`
-	StoragePoolUri         utils.Nstring          `json:"storagePoolUri,omitempty"`
-	StorageSystemUri       utils.Nstring          `json:"storageSystemUri,omitempty"`
-	ProvisionType          string                 `json:"provisionType,omitempty"`
-	ProvisionedCapacity    string                 `json:"provisionedCapacity,omitempty"`
-	ProvisioningParameters ProvisioningParameters `json:"provisioningParameters,omitempty"`
+	Category                  string                    `json:"category,omitempty"`
```

Comment:
is this struct backward compatible with API300? if not, then lets create a new file for new struct.